### PR TITLE
Removing cvx_kron unused function.

### PR DESCRIPTION
**Pull request by vprusso**
- As `cvx_kron` is no longer used, am removing.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@ build/
 coverage.xml
 jupyter_execute/
 venv/*
+.aider*
```
Comment (purva-thakre) [inline .gitignore:20]: What is this new addition related to?

**Review by purva-thakre — APPROVED** [review]
LGTM! Thanks for noticing that we are not using this function anymore.

File: .gitignore (hunk 1 above)
Comment (vprusso) [reply .gitignore]: Ah, that was some VSCode extension I was experimenting with. I removed that from the `.gitignore` file.
